api/src/main/java/org/pmiops/workbench/api/ProfileController.java

```diff
@@ -22,6 +26,12 @@
   }
 
   @Override
+  public ResponseEntity<List<BillingProjectMembership>> billing() {
```

Comment:
Is the idea that this is a getter for billing projects? 
If so, what do you think about changing the signature to `getBilling()` (to match the `getMe()` endpoint)

Reply:
Good catch, renamed to getBillingProjects()